protoc-artifacts/README.md

```diff
@@ -54,7 +54,62 @@ Before you can upload artifacts to Maven Central repository, make sure you have
 read [this page](http://central.sonatype.org/pages/apache-maven.html) on how to
 configure GPG and Sonatype account.
 
-Use the following command to upload artifacts:
+You need to perform the deployment for every platform that you want to
+suppport. DO NOT close the staging repository until you have done the
```

Comment:
Done.
The only cross-compilation that I have tested is building 32-bit artifact on 64-bit systems. Theoretically we can build Windows artifacts on Linux with the help of MinGW, but I have not tested it.